classes/ObjectModel.php

```diff
@@ -1149,27 +1149,39 @@ public function validateField($field, $value, $id_lang = null, $skip = [], $huma
 
         // Check field validator
         if (!in_array('validate', $skip) && !empty($data['validate'])) {
-            if (!method_exists('Validate', $data['validate'])) {
-                throw new PrestaShopException($this->trans('Validation function not found: %s.', [$data['validate']], 'Admin.Notifications.Error'));
+            $callable = $data['validate'];
+            if (!is_callable($callable)) {
+                $callable = 'Validate::'. $data['validate'];
+            }
+
+            if (!is_callable($callable)) {
+                throw new PrestaShopException(
+                    $this->trans(
+                        'Validation function not found: %s.',
+                        array($data['validate']),
+                        'Admin.Notifications.Error'
+                    )
+                );
             }
 
             if (!empty($value)) {
-                $res = true;
-                if (Tools::strtolower($data['validate']) == 'iscleanhtml') {
+                $is_valid = true;
```

Comment:
```suggestion
                $isValid = true;
```

Reply:
I disagree with this as this forces me to update all the file: this is not the intent of my contribution, let's forcus on the feature and let a bot clean variables, please.

Reply:
You have renamed $res to $is_valid yourself ;)

Reply:
No problem, let's re introduce $res !